examples/experimental/torch/classification/configs/inception_v3_imagenet_int8.json

```diff
@@ -0,0 +1,37 @@
+{
+    "model": "inception_v3",
+    "pretrained": true,
+    "input_info": {
+      "sample_size": [2, 3, 299, 299]
+    },
+    "num_classes": 1000,
+    "batch_size" : 512,
+    "epochs": 1,
+    "multiprocessing_distributed" : true,
+    "optimizer": {
+        "type": "Adam",
+        "base_lr": 1e-5,
+        "schedule_type": "multistep",
+        "steps": [
+            1
+        ]
+    },
+    "compression": {
+        "algorithm": "quantization",
+        "overflow_fix": "disable",
```

Comment:
why `overflow_fix` be `disable` for this config?

Reply:
Enabling `overflow_fix` implicitly changes weight bit-widths from `int8` to `int7` to prevent `w1 * a1 + w2 * a2` from overflowing at `int16` precision. It means that the model size is implicitly compressed 7/8 already if we enable `overflow_fix`. Therefore, our compression rate measurement will be inaccurate and it will make comparison between `M8A8` and `W8A8` unfaithful.